cordformation/src/main/kotlin/net/corda/plugins/Node.kt

```diff
@@ -56,14 +57,24 @@ open class Node @Inject constructor(private val project: Project) {
         private set
     private var rpcSettings: RpcSettings = RpcSettings()
     private var webserverJar: String? = null
-    private var p2pPort = 10002
+    internal var p2pPort = 10002
```

Comment:
Probably want this as `private set` as well (basically saying that only this class is allowed to change the value).

Reply:
The `Node` object is exposed into people's `build.gradle` files and can be accessed via Groovy, so "yes" - "Lock it down!"